sidekick/lib/src/util/name_suggester.dart

```diff
@@ -19,22 +19,23 @@ class NameSuggester {
         .sortedBy((it) => it.length);
     final names = [
       ...suggestions,
-      'Enter your own',
+      '<Enter your own>',
     ];
 
     final String answer = dcli.menu(
       options: names,
-      prompt: 'Type the number of the fitting acronym or choose Enter your own',
+      prompt: 'Select cliName:',
```

Comment:
For consistency, further down we print `Enter your CLI name:`
```suggestion
      prompt: 'Select CLI name:',
```